source/extensions/clusters/redis/redis_cluster.cc

```diff
@@ -225,22 +227,14 @@ RedisCluster::RedisDiscoverySession::RedisDiscoverySession(
 // Convert the cluster slot IP/Port response to and address, return null if the response
 // does not match the expected type.
 Network::Address::InstanceConstSharedPtr
-RedisCluster::RedisDiscoverySession::RedisDiscoverySession::ProcessCluster(
+RedisCluster::RedisDiscoverySession::RedisDiscoverySession::processClusterByIP(
```

Comment:
Maybe something like `ipAddresFromClusterEntry` or something like that? `process` is a bit ambiguous